src/modules/EditUserProfile/components/EditUserProfileUI/EditUserProfileUIView.tsx

```diff
@@ -0,0 +1,135 @@
+        <InputLabel>
+          {stringSet.EDIT_PROFILE__IMAGE_LABEL}
+        </InputLabel>
+        <div className="sendbird-edit-user-profile__img__avatar">
+          <Avatar
+            width="80px"
+            height="80px"
+            src={currentImg || user?.profileUrl}
+          />
+        </div>
+        <input
+          ref={hiddenInputRef}
+          type="file"
+          accept="image/gif, image/jpeg, image/png"
+          style={{ display: 'none' }}
+          onChange={(e) => {
+            setCurrentImg(URL.createObjectURL(e.target.files[0]));
+            setProfileImage(e.target.files[0]);
+            hiddenInputRef.current.value = '';
+          }}
+        />
+        <TextButton
+          className="sendbird-edit-user-profile__img__avatar-button"
+          disableUnderline
+          onClick={() => hiddenInputRef.current.click()}
+        >
+          <Label
+            type={LabelTypography.BUTTON_1}
+            color={LabelColors.PRIMARY}
+          >
+            {stringSet.EDIT_PROFILE__IMAGE_UPLOAD}
+          </Label>
+        </TextButton>
+      </section>
+      <section className="sendbird-edit-user-profile__name">
+        <InputLabel>
+          {stringSet.EDIT_PROFILE__NICKNAME_LABEL}
+        </InputLabel>
+        <Input
+          required={user?.nickname !== ''}
+          name="sendbird-edit-user-profile__name__input"
+          ref={inputRef}
+          value={user?.nickname}
+          placeHolder={stringSet.EDIT_PROFILE__NICKNAME_PLACEHOLDER}
+        />
+      </section>
+      <section className="sendbird-edit-user-profile__userid">
+        <InputLabel>
+          {/*  userID */}
+          {stringSet.EDIT_PROFILE__USERID_LABEL}
+        </InputLabel>
+        <Input
+          disabled
+          name="sendbird-edit-user-profile__userid__input"
+          value={user?.userId}
+        />
+      </section>
+      <section className="sendbird-edit-user-profile__theme">
+        <InputLabel>
+          {stringSet.EDIT_PROFILE__THEME_LABEL}
+        </InputLabel>
+        <div className="sendbird-edit-user-profile__theme__theme-icon">
+          {
+            theme === 'dark'
+              ? (
+                <Icon
+                  onClick={() => {
+                    setCurrentTheme('light');
+                    onThemeChange?.('light');
+                    // if (onThemeChange && typeof onThemeChange === 'function') {
+                    //   onThemeChange('light');
+                    // }
+                  }}
+                  type={IconTypes.TOGGLE_ON}
+                  width={44}
+                  height={24}
+                />
+              )
+              : (
+                <Icon
+                  onClick={() => {
+                    setCurrentTheme('dark');
+                    onThemeChange?.('dark');
+                    // if (onThemeChange && typeof onThemeChange === 'function') {
+                    //   onThemeChange('dark');
+                    // }
```

Comment:
If we are going to keep comments, we should add explanation too. Otherwise lets remove it!